android/settings.gradle

```diff
@@ -9,3 +9,6 @@ if (settings.hasProperty("newArchEnabled") && settings.newArchEnabled == "true")
     include(":ReactAndroid:hermes-engine")
     project(":ReactAndroid:hermes-engine").projectDir = file('../node_modules/react-native/ReactAndroid/hermes-engine')
 }
+
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```

Comment:
According to the [docs](https://www.npmjs.com/package/react-native-vector-icons), these lines are optional and are only necessary if you intend to utilize the `Icon.getImageSource` function.
But I don't see you used it.
What was the reason for adding those lines?

Reply:
My bad here, I was testing something that wasn't working, thank you.